CMakeLists.txt

```diff
@@ -88,10 +88,25 @@ macro(jerry_add_compile_flags)
   jerry_add_flags(CMAKE_C_FLAGS ${ARGV})
 endmacro()
 
+# SHOULD THIS BE, SAY, AT THE TOP OF THE FILE?
+if(CMAKE_C_COMPILER_ID MATCHES "GNU")
```

Comment:
I'm not sure that this condition is necessary, maybe you should use CMAKE_COMPILER_IS_GNUCC to check if the compiler is a variant of gcc.